pkg/git/git.go

```diff
@@ -9,20 +9,26 @@ import (
 	"github.com/craiggwilson/goke/task"
 )
 
-// Branch returns the current branch of the git repository in the current working directory.
-func Branch(ctx *task.Context) (string, error) {
-	cmd := exec.CommandContext(ctx, "git", "rev-parse", "--abbrev-ref", "HEAD")
+func runGitCmd(ctx *task.Context, name string, args ...string) (string, error) {
```

Comment:
This is actually quite general... does a helper like this exist already somewhere in Goke I can use instead? Should I move this to somewhere more general?